manifests/handler.pp

```diff
@@ -67,6 +67,11 @@
 #   Default: false.
 #   Valid values: true, false
 #
+# [*handle_silenced*]
```

Comment:
Please add an example to https://github.com/sensu/sensu-puppet/blob/master/tests/sensu-server.pp and document which file should be modified what json should be expected. That way we can functionally test this behaves as expected.